azure_arc_k8s_jumpstart/aro/arm_template/scripts/az_connect_aro.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+# <--- Change the following environment variables according to your Azure service principal name --->
+
+echo "Exporting environment variables"
+export appId='<Azure service principal Id>'
+export password='<Azure service principal password>'
+export resourceGroup='<Azure Resource Group name>'
+export clusterName='<The name of your Aro cluster as it will be shown in Azure Arc>'
```

Comment:
See if you can align to vars names like in [here](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/cluster_api/capi_azure/installCAPI.sh)